Sources/Penny/EvolutionChecker.swift

```diff
@@ -246,29 +257,25 @@ actor EvolutionChecker {
                 description: """
                 > \(summary)
                 \(status)
+                \(upcomingFeatureFlag)
                 \(authorsString)
-                \(reviewManagerString)
+                \(reviewManagersString)
                 """,
                 url: proposalLink,
                 color: proposal.status.state.color
             )],
-            components: await makeComponents(proposal: proposal, proposalLink: proposalLink)
+            components: await makeComponents(proposal: proposal)
         )
     }
 
-    private func makeComponents(
-        proposal: Proposal,
-        proposalLink: String?
-    ) async -> [Interaction.ActionRow] {
+    private func makeComponents(proposal: Proposal) async -> [Interaction.ActionRow] {
         var buttons: [Interaction.ActionRow] = [[]]
 
-        if let proposalLink,
-           let link = await findForumPostLink(link: proposalLink) {
-            let description = link.description.trimmingCharacters(in: .punctuationCharacters).capitalized
+        if let discussion = proposal.discussions?.last {
             buttons[0].components.append(
                 .button(.init(
-                    label: "\(description) Post",
-                    url: link.destination
+                    label: "\(discussion.name.capitalized) Post",
+                    url: discussion.link
                 ))
             )
         }
```

Comment:
The new JSON also contains links to discussions, we were previously parsing these manually with some help from swift-markdown.